plotsearch/serializers/plot_search.py

```diff
@@ -867,6 +867,7 @@ class AreaSearchSerializer(EnumSupportSerializerMixin, serializers.ModelSerializ
         queryset=User.objects.all(),
         related_serializer=UserSerializer,
         required=False,
+        allow_null=True,
```

Comment:
Do we need migration for changing field to allow nulls?

Reply:
Very valid question, in this case no, since this is a serializer. Migrations apply for models